woostarter/patterns/social-media-call-to-action-with-hero-image.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Title: Social media call to action with hero image
+ * Slug: woostarter/social-media-call-to-action-with-hero-image
+ * Categories: Banners, Call to Action
+ */
+?>
+<!-- wp:group {"metadata":{"name":"Social media call to action with hero image"},"align":"full","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull"><!-- wp:cover {"url":"<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/hero-image.webp","dimRatio":50,"customOverlayColor":"#3d3d3e","isUserOverlayColor":false,"minHeight":60,"minHeightUnit":"vh","sizeSlug":"large","align":"full","layout":{"type":"constrained"}} -->
+<div class="wp-block-cover alignfull" style="min-height:60vh"><img class="wp-block-cover__image-background  size-large" alt="" src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/hero-image.webp" data-object-fit="cover"/><span aria-hidden="true" class="wp-block-cover__background has-background-dim" style="background-color:#3d3d3e"></span><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","fontSize":"x-large"} -->
+<p class="has-text-align-center has-x-large-font-size"><?php esc_html_e('Follow us on Instagram', 'woostarter');?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"align":"center","style":{"typography":{"fontStyle":"normal","fontWeight":"300"},"elements":{"link":{"color":{"text":"var:preset|color|theme-1"}}}},"textColor":"theme-1","fontSize":"medium"} -->
+<p class="has-text-align-center has-theme-1-color has-text-color has-link-color has-medium-font-size" style="font-style:normal;font-weight:300"><?php /* Translators: 1. is the start of a 'a' HTML element, 2. is the end of a 'a' HTML element */ 
+echo sprintf( esc_html__( '%1$s@example%2$s', 'woostarter' ), '<a href="' . esc_url( 'https://www.instagram.com/' ) . '">', '</a>' ); ?></p>
```

Comment:
This is a real account! Maybe we should link to the woo account?

Reply:
mmh, but we are not linking to the account. I'm happy to change the label but I think maybe the link we should be careful about: https://make.wordpress.org/themes/handbook/review/required/#13-selling-credits-links-and-spam

Reply:
I removed the link altogether, I think this is fine to have